core/controllers/editor_test.py

```diff
@@ -697,6 +702,73 @@ def test_deletion_rights_for_published_exploration(self):
         self.assertEqual(response.status_int, 200)
         self.logout()
 
+    def test_logging_info_after_deletion(self):
+        """Tests correctness of logged statements while deleting exploration"""
+        logging_info = []
+
+        def add_logging_info(msg, *_):
+            if msg != 'all_pending: clear %s':
```

Comment:
This message is logged by function clear_all_pending() in oppia_tools/google_appengine_1.9.19/google_appengine/google/appengine/ext/ndb/tasklets.py
I think this is used to clear all request from the task queue (i am not sure though)

Since, this message doesn't always occur I filtered out in if statement.